src/main/plugins/org.dita.pdf2/cfg/common/vars/ar.xml

```diff
@@ -328,16 +328,46 @@ See the accompanying license.txt file for applicable licenses.
 
     <!-- The header that appears on the even-numbered glossary pages. -->
     <variable id="Glossary even header"><param ref-name="pagenum"/> | <param ref-name="heading"/> | <param ref-name="prodname"/></variable>
-     
+
      <!-- The heading string to put at the top of the Glossary -->
      <variable id="Glossary">قاموس المصطلحات</variable>
-     
+
      <!-- The heading string to put at the top of the List of Tables -->
      <variable id="List of Tables">الجداول</variable>
-     
+
      <!-- The heading string to put at the top of the List of Figures -->
      <variable id="List of Figures">الأشكال</variable>
 
+     <!-- The heading string to put at the top of the List of Changes -->
+     <variable id="List of Changes"></variable>
```

Comment:
@robander how do we want to handle untranslated strings in variable files. Having an English string in Arabic strings is not a particular good if the user doesn't speak Arabic.